src/ngx_http_lua_directive.c

```diff
@@ -1700,6 +1701,71 @@ ngx_http_lua_conf_read_lua_token(ngx_conf_t *cf,
 }
 
 
+char *
+ngx_http_lua_intercept_error_log(ngx_conf_t *cf, ngx_command_t *cmd,
```

Comment:
I think this function should go to the `ngx_http_lua_log.c` file instead. This `util.c` file is already too big. Better not make it grow.

Also, I don't think this function should even exist when `HAVE_INTERCEPT_ERROR_LOG_PATCH` is not defined.